microprofile-jwt-auth-tool/src/main/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/RsaKeyTool.java

```diff
@@ -99,7 +100,7 @@ public PrivateKey getPrivateKey() {
 
     private void prepare(final URI pkcs8Key) {
         try (final FileReader fileReader = new FileReader(new File(pkcs8Key));
-             final BufferedReader bufferedFileReader = new BufferedReader(fileReader)) {
+                final BufferedReader bufferedFileReader = new BufferedReader(fileReader)) {
```

Comment:
Speaking of readability, this is not intuitive at all. Conforming to what I've seen in previous reviews here, please consider how to improve it.

Reply:
This is an automated adjustment to the code.

Please create PR with a change to https://github.com/jboss-eap-qe/eap-microprofile-test-suite/blob/master/ide-config/eclipse-format.xml if you want to change this.

Reply:
Hi @rsvoboda, sorry but I don't think your comment - and action - is appropriate:
1. You did file a PR to improve code readability, I guess. This is what formatting is all about. So you are responsible for the PR and consequently should consider input that others have to consider when filing PRs
2. You followed up to my comment by approving and merging the PR on your own. IMO you should at least have asked another review from a peer before doing that.

Reply:
@fabiobrz please see https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/108

These changes were generated simply by running `mvn clean install -DskipTests -DskipITs -Djboss.home=foo`.

I didn't do any manual changes to the code. Somebody just ignored/forgot `Code is formatted, imports ordered, code compiles and tests are passing` checkbox for PR.

There is no need for another review of this change. It's just fixing our current code formatting

Reply:
@rsvoboda ok thanks.